File: Project1/ClientApp/src/app/project/models/task/task-update-query.ts
import { TaskStateEnum } from "./task-state-enum";

export class TaskUpdateQuery {
  name: string;
  id: string;
  description: string;
  projectId: string;
  state: TaskStateEnum;
  date: Date;
  everyday: boolean;
  estimated: number;
  completed: number;
  rest: number;
  priority: number;
}

File: Project1/Controllers/TaskElement/Model/TaskUpdateQuery.cs
using Amirez.AmipBackend.Controllers.Generic.Model;
using System;

namespace Amirez.AmipBackend.Controllers.TaskElement.Model
{
    public class TaskUpdateQuery : UpdateBaseQuery
    {
        public string Description { get; set; }

        public int Estimated { get; set; }

        public int Completed { get; set; }

        public int Rest { get; set; }

        public Guid ProjectId { get; set; }
        public int Priority { get; set; }
    }
}
